jmclient/jmclient/blockchaininterface.py

```diff
@@ -203,19 +333,16 @@ def __init__(self, jsonRpc, network, wallet_name):
                     "setting in joinmarket.cfg) instead. See docs/USAGE.md "
                     "for details.")
 
-    def is_address_imported(self, addr):
+    def is_address_imported(self, addr: str) -> bool:
         return len(self._rpc('getaddressinfo', [addr])['labels']) > 0
 
-    def get_block(self, blockheight):
+    def get_block(self, blockheight: int) -> Optional[str]:
         """Returns full serialized block at a given height.
         """
         block_hash = self.get_block_hash(blockheight)
-        block = self._rpc('getblock', [block_hash, False])
-        if not block:
-            return False
-        return block
+        return self._rpc('getblock', [block_hash, 0])
 
-    def _rpc(self, method, args):
+    def _rpc(self, method: str, args: Optional[list] = None):
```

Comment:
Return type typehint?

Reply:
I'm not quite sure about possible return types here. It depends on what RPC calls to Bitcoin Core return and can be anything and that can change any time beyond our control?

Will check about `args`. Probably should be changed to `args: list = []`.

Reply:
I think putting `Any` is fine if we are not sure. It should also be easier to look for these cases and eventually fix them in the future.

Reply:
Addressed in 15c099a51e54f6e8666e07affceb208942ceb2ba